packages/ui/src/core/textarea/textarea.props.ts

```diff
@@ -22,9 +22,10 @@ export interface TextareaProps extends React.HTMLAttributes<HTMLTextAreaElement>
   rows?: number;
   showActionButton?: boolean;
   showClearButton?: boolean;
-  showCopyIcon?: boolean;
+  hideCopyIcon?: boolean;
   actionButtonLabel?: string;
   clearAllButtonLabel?: string;
+  onChange?: (event: ChangeEvent<HTMLTextAreaElement>) => void;
```

Comment:
you can remove this as we don't need it because it is already inside React.HTMLAttributes<HTMLTextAreaElement> interface